README.md

```diff
@@ -6,13 +6,32 @@ https://registry.terraform.io/providers/mattmoor/cosign
 
 ## Usage
 
-This provides an `cosign_sign` resource that will sign the provided image digest
-with `cosign`.
+This provides an `cosign_sign` and `cosign_verify` resources that will sign and
+verify the provided images with `cosign`.
 
 ```hcl
 provider "cosign" {}
 
+# Verify the Chainguard base image against a policy from
+# github.com/sigstore/policy-controller.
+resource "cosign_verify" "example" {
+  image  = "cgr.dev/chainguard/static:latest-glibc"
```

Comment:
WDYT about requiring this to be by digest?

Reply:
Sorta 👎 , lemme explain.

I feel like tags are ok when combined with policy because you still get cryptographic guarantees about the image.  Also, while there are still reasons to verify policies against digests (e.g. build horizon!), you presumably established some level of trust when you resolved the digest in the first place, so this provides the _**most**_ value for tags.

Also, without a rule to compose with to resolve a digest, users can't use this to verify `ko` base images OOTB, which feels like a key feature.

Reply:
https://github.com/imjasonh/terraform-provider-crane/blob/main/docs/data-sources/ref.md